cli/src/cli_util.rs

```diff
@@ -3298,3 +3298,48 @@ fn format_template_aliases_hint(template_aliases: &TemplateAliasesMap) -> String
     );
     hint
 }
+
+pub fn parse_author_arg(author: &str) -> Result<(String, String), clap::Error> {
```

Comment:
nit: we don't have to use `clap::Error`. `&'static str` or `String` also works.

`text_util` or `description_util` might be better place to host this function. `cli_util` is bloated. We have `split_email()` in `text_util.rs` btw.

Reply:
Right, I've got so used to `anyhow`/`thiserror` in recent years I tried to create `&'static str` error but with one `Err()` too many over the weekend.
Changed to `&'static str`.